docs/howtos/migrate-swagger/migrate-arm-tips.md

```diff
@@ -0,0 +1,228 @@
+interface Widgets {
+  createOrUpdate is ArmResourceCreateOrReplaceAsync<Widget>;
+}
+```
+
+The name of the request body parameter is `resource` so you can change the name in clients using an augment decorator
+
+```tsp
+@@projectedName(Widgets.createOrUpdate::parameters.resource,
+  "client",
+  "<desired-request-body-parameter-name>"
+);
+```
+
+Note that this works for _any_ PUT operation template.
+
+#### Update (PATCH) APIs
+
+Given a PATCH operation, for example:
+
+```tsp
+interface Widgets {
+  update is ArmResourcePatchAsync<Widget, WidgetProperties>;
+}
+```
+
+The name of the request body parameter is `properties` so you can change the name in clients using an augment decorator
+
+```tsp
+@@projectedName(Widgets.update::parameters.properties,
+  "client",
+  "<desired-request-body-parameter-name>"
+);
+```
+
+Note that this works for _any_ PATCH operation template.
+
+### Action (POST) APIs
+
+Given a POST operation, for example:
+
+```tsp
+interface Widgets {
+  mungeWidget is ArmResourceActionAsync<Widget, MungeRequest, MungeResponse>;
+}
+```
+
+The name of the request body parameter is `body` so you can change the name in clients using an augment decorator
+
+```tsp
+@@projectedName(Widgets.mungeWidget::parameters.body,
+  "client",
+  "<desired-request-body-parameter-name>"
+);
+```
+
+Note that this works for _any_ POST operation template.
+
+### Adding Request Query or Header Parameters
+
+The `Parameters` template parameter allows you to specify additional parameters after the operation path (for example, query and header parameters) in the form of a model, with each model property corresponding to a parameter. You may use intersection to combine multiple separate parameters.
+
+```tsp
+// all list query params
+listBySubscription is ArmListBySubscription<Widget, Parameters = Azure.Core.StandardListQueryParameters>;
+
+// intersecting individual parameters
+listBySubscription is ArmListBySubscription<Widget, Parameters =  Azure.Core.TopQueryParameter & Azure.Core.SkipQueryParameter>;
+```
+
+### Changing Response Types
+
+The `Response` parameter allows you to specify non-error responses to the operation.
+
+```tsp
+// all list query params
+listBySubscription is ArmListBySubscription<Widget, Response = MyCustomCollectionType>;
+```
+
+### Changing Error Types
+
+The `Error` parameter allows you to change the default error type used in an operation.
+
+```tsp
+// all list query params
+listBySubscription is ArmListBySubscription<Widget, Error = MyCustomError>;
+```
+
+### Converting Synchronous Operations to LROs
+
+You can generally choose an asynchronous operation template that matches your operation.
+
+#### Templates for Async PUT Operations
+
+- `ArmCreateOrReplaceAsync` is a PUT operation that uses the 'resource' definition in the request body, and return a `200` response and a `201` response, both of which contain the created/updated resource in the response payload. The 201 response contains 'Location` LRO header.
```

Comment:
correction needed:  ArmResourceCreateOrReplaceSync

Reply:
No, this is describing both of the async operations